frontend/src/pages/RunList.tsx

```diff
@@ -488,29 +468,8 @@ class RunList extends React.PureComponent<RunListProps, RunListState> {
           'Failed to get associated pipeline version: ' + (await errorToMessage(err));
         return;
       }
-    } else if (pipelineId) {
-      // For backward compatibility. Runs created before version is introduced
-      // refer to pipeline id instead of pipeline version id.
-      let pipelineName = RunUtils.getPipelineName(displayRun.run);
-      if (!pipelineName) {
-        try {
-          const pipeline = await Apis.pipelineServiceApi.getPipeline(pipelineId);
-          pipelineName = pipeline.name || '';
-        } catch (err) {
-          displayRun.error = 'Failed to get associated pipeline: ' + (await errorToMessage(err));
-          return;
-        }
-      }
-      displayRun.pipelineVersion = {
-        displayName: pipelineName,
-        pipelineId,
-        usePlaceholder: false,
-        versionId: undefined,
-      };
```

Comment:
Why are this part deleted?

Reply:
Because `pipelineId` is no longer store in `pipeline_spec` or other field in `run` when using v2 API. That is, we are not able to get `pipelineId` from getRun or listRun response.